### PR TITLE
MOBILE-1140: In Bank selection screen show also those banks, which account linking is not supported

### PR DESCRIPTION
**Pull request by PocketDaniel**

### CODE REVIEW

File: Sources/Kevin/Accounts/KevinAccountsConfiguration.swift [hunk 1]
```diff
@@ -11,24 +11,33 @@ import Foundation
...
         self.callbackUrl = callbackUrl
+        self.showUnsupportedBanks = showUnsupportedBanks
     }
     
     public class Builder {
         
         private let callbackUrl: URL
-        
-       public init(callbackUrl: URL) {
+        private let showUnsupportedBanks: Bool
```
Comment (edgar-zigis) [inline Sources/Kevin/Accounts/KevinAccountsConfiguration.swift:27]: Builder should not have a constructor parameter with `showUnsupportedBanks` as only `callbackUrl` is mandatory. Provide this param in the builder instead like Android does [here](https://github.com/getkevin/kevin-android/pull/80/files). Default value for `showUnsupportedBanks` should be false.

File: Sources/Kevin/Accounts/BankSelection/KevinBankSelectionConfiguration.swift [hunk 2]
```diff
@@ -16,20 +16,23 @@ internal struct KevinBankSelectionConfiguration {
...
     
     public init(
         selectedCountry: KevinCountry,
         isCountrySelectionDisabled: Bool,
         countryFilter: Array<KevinCountry>,
         selectedBankId: String?,
         authState: String,
-        exitSlug: String
+        exitSlug: String,
+        showOnlyAccountLinkingSupportedBanks: Bool
```
Comment (edgar-zigis) [inline Sources/Kevin/Accounts/BankSelection/KevinBankSelectionConfiguration.swift:28]: I think we need to rename this to `excludeBanksWithoutAccountLinkingSupport`

File: Sources/Kevin/Accounts/KevinAccountsPlugin.swift [hunk 3]
```diff
@@ -32,6 +32,13 @@ public class KevinAccountsPlugin: KevinPlugin {
         return configuration.callbackUrl
     }
     
+    public func isShowUnsupportedBanks() throws -> Bool {
```
Comment (edgar-zigis) [inline Sources/Kevin/Accounts/KevinAccountsPlugin.swift:35]: `shouldExcludeBanksWithoutAccountLinkingSupport()` and also change condition then

**Review by edgar-zigis — CHANGES_REQUESTED** [review]

File: Sources/Kevin/Accounts/LinkingSession/KevinAccountLinkingSession.swift [hunk 4]
```diff
@@ -80,7 +80,11 @@ final public class KevinAccountLinkingSession {
                     }
                 }
             } else {
-                delegate?.onKevinAccountLinkingStarted(controller: initializeBankSelection(configuration: configuration))
+                do {
+                    delegate?.onKevinAccountLinkingStarted(controller: try initializeBankSelection(configuration: configuration))
+                } catch {
+                    self.delegate?.onKevinAccountLinkingCanceled(error: error)
```
Comment (typuk) [inline Sources/Kevin/Accounts/LinkingSession/KevinAccountLinkingSession.swift:86]: can omit self

**Review by edgar-zigis — APPROVED** [review]

**Pull request merged**